lib/LANraragi/Controller/Batch.pm

```diff
@@ -33,7 +33,7 @@ sub index {
         if (-e $zipfile) {
             $arclist .=
                 "<li><input type='checkbox' name='archive' id='$id' class='archive' >"
-                . "<label for='$id'> $title</label></li>";
+                . "<label for='$id'> <a href='./edit?id=$id' style='text-decoration: none;'>$title</a></label></li>";
```

Comment:
From a UX perspective it's super weird here that the checkbox and label don't perform the same action when clicking on them, I think a button on the side would be more appropriate.  

Also, putting <a> tags in the checklist breaks colors on both the Yotsugi and Nadeko themes.